src/framework/ui/internal/platform/linux/linuxplatformthemelistener.h

```diff
@@ -0,0 +1,42 @@
+/*
+ * SPDX-License-Identifier: GPL-3.0-only
+ * MuseScore-CLA-applies
+ *
+ * MuseScore
+ * Music Composition & Notation
+ *
+ * Copyright (C) 2023 MuseScore BVBA and others
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+#ifndef MU_UI_LINUXPLATFORMTHEMELISTENER_H
+#define MU_UI_LINUXPLATFORMTHEMELISTENER_H
+
+#include "internal/iplatformtheme.h"
+
+class QDBusVariant;
+
+namespace mu::ui {
+class LinuxPlatformThemeListener : public QObject
+{
+    Q_OBJECT
+public slots:
+    void processSettingChange(const QString& group, const QString& key, const QDBusVariant& value);
```

Comment:
I think we don't need to create a separate class for this slot
You can add this slot to LinuxPlatformTheme

Reply:
> I think we don't need to create a separate class for this slot You can add this slot to LinuxPlatformTheme

`LinuxPlatformTheme` would then need to inherit from QObject. Is that okay?

Reply:
@adazem009 I believe that if you use the modern `connect` overloads (for example, passing `&LinuxPlatformTheme:: processSettingChange` instead of `SLOT(processSettingChange(…))` for the slot, and the same for the signal), then `&LinuxPlatformTheme:: processSettingChange` doesn't even need to be a slot, and `LinuxPlatformTheme` doesn't need to be a `QObject`. Not 100% sure though.

Reply:
> @adazem009 I believe that if you use the modern `connect` overloads (for example, passing `&LinuxPlatformTheme:: processSettingChange` instead of `SLOT(processSettingChange(…))` for the slot, and the same for the signal), then `&LinuxPlatformTheme:: processSettingChange` doesn't even need to be a slot, and `LinuxPlatformTheme` doesn't need to be a `QObject`. Not 100% sure though.

The problem is that `QDBusConnection::connect()` doesn't have the modern overloads. (https://doc.qt.io/qt-5/qdbusconnection.html#connect)

See https://forum.qt.io/topic/75117/lambdas-in-qdbusconnection-connect

Reply:
> I think we don't need to create a separate class for this slot You can add this slot to LinuxPlatformTheme

done